scripts/terminalserver/terminalserver.jl

```diff
@@ -159,8 +179,7 @@ end
 
 run(conn_endpoint)
 
-@async begin
-
+@async try
```

Comment:
What's the point of this? The REPL connection is effectively dead after the `catch` anyways, no?

Reply:
There is at least an error message shown in the REPL, otherwise the error just silently disappears. What we should really do is put a crash report into the catch block, I'll do that in a new PR once this one is merged (it requires a few more things).